janusgraph-core/src/main/java/org/janusgraph/diskstorage/configuration/ModifiableConfiguration.java

```diff
@@ -60,6 +63,15 @@ public void freezeConfiguration() {
         if (!isFrozen()) setFrozen();
     }
 
+    private boolean upgradeFixed(String name) {
+        for (String value : UPGRADEABLE_FIXED) {
```

Comment:
1. This loop isn't needed because `List` has a method `contains`. 
2. You should use `HashSet` to make complexity of `upgradeFixed` be `O(1)` and not `O(n)`.
3. It would be clearer if the name of checking method would start from `is` or `has`.